blackbox/docs/release_notes/1.2.0.txt

```diff
@@ -188,4 +193,79 @@ Fixes
 
       SELECT count(*) from t1, t2 order by 1
 
+
+Upgrade Guide
+=============
+
+Daemon User
+-----------
+
+You can no longer run CrateDB as the superuser on Unix-like systems. You should
```

Comment:
calling the section "upgrade guide" and then starting to write "you can no longer" is somehow odd.  either you rename it to "Upgrade Notes" which i would prefer, it works - but if it is called guide it would mean i have to write the paras like: "ensure that your crate daemon is not running as the root user, since ..."